CHANGELOG.md

```diff
@@ -6,6 +6,23 @@
 
 ## [Unreleased] - ReleaseDate
 
+### Added
+
+- [#16](https://github.com/embedded-graphics/tinybmp/pull/16) Added `display` example to display TGA images using the embedded-graphics simulator.
```

Comment:
Did we discuss long ago in the e-g repo that changes like this don't make much sense in the changelog?

Reply:
Yes, we did and I removed the entry from the changelog.